components/pages/Applications/Dashboard/Applications/InProgress/ButtonGroup.tsx

```diff
@@ -84,6 +85,14 @@ const getButtonConfig = (
           icon: icons.user,
         },
       ];
+      case ApplicationState.PAUSED:
+        return [
+          {
+            content: 'Complete Attestation',
+            link,
+            icon: icons.calendar,
+          },
+        ];
     case ApplicationState.CLOSED:
       [
```

Comment:
unrelated to your PR, but this line is missing a `return` to match the other cases.
May be a good thing to fix while at it.
```suggestion
      return [
```

Reply:
`return` added in line 98 to make the case work properly.